tests/performance-tests/src/utils/publicApiService.ts

```diff
@@ -0,0 +1,290 @@
+import { RefinedResponse } from 'k6/http';
+import HttpClient from './httpClient';
+
+const applicationJsonHeaders = {
+  'Content-Type': 'application/json',
+};
+
+export const datSetQueryNodeTypes = ['criteria', 'and', 'or', 'not'] as const;
+
+export type DataSetQueryNodeType = typeof datSetQueryNodeTypes[number];
+
+export const dataSetQueryIdOperators = ['eq', 'notEq', 'in', 'notIn'] as const;
+
+export type DataSetQueryIdOperator = typeof dataSetQueryIdOperators[number];
+
+export const dataSetQueryComparableOperators = [
+  'eq',
+  'notEq',
+  'in',
+  'notIn',
+  'gt',
+  'gte',
+  'lt',
+  'lte',
+] as const;
+
+export type DataSetQueryComparableOperator = typeof dataSetQueryComparableOperators[number];
+
+export interface Publication {
+  id: string;
+  title: string;
+  slug: string;
+}
+
+interface DataSet {
+  id: string;
+  name: string;
```

Comment:
This is `title` now

Reply:
:+1: Aha good to know thanks - yeah makes sense to be title!

Reply:
Done ta - renamed DATA_SET_NAMES parameter to be DATA_SET_TITLES also.